src/Arcus.Messaging.Tests.Integration/MessagePump/ServiceBusMessagePumpDockerTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Arcus.EventGrid.Parsers;
+using Arcus.EventGrid.Testing.Infrastructure.Hosts.ServiceBus;
+using Arcus.Messaging.ServiceBus.Core.Extensions;
+using Arcus.Messaging.Tests.Core.Events.v1;
+using Arcus.Messaging.Tests.Core.Generators;
+using Microsoft.Azure.EventGrid.Models;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Messaging.Tests.Integration.MessagePump
+{
+    [Trait("Category", "Docker")]
+    public class ServiceBusMessagePumpDockerTests : IntegrationTest, IAsyncLifetime
```

Comment:
Can't we re-use these for the other tests as well?

Reply:
You mean the encoding? Or you mean the test message service?

Reply:
No the whole test range as they are pretty overlapping, no?

Reply:
Not sure what you're getting at. The difference with these Docker tests is that they require external resources, while other tests don't. You want to extract simulair functionality or so?

Reply:
👌 just wanted to check, it's ok